dynamic_models/schema.py

```diff
@@ -14,8 +15,15 @@ def update_table(self, new_model):
         self.initial_model = new_model
 
     def create_table(self, new_model):
-        with connection.schema_editor() as editor:
-            editor.create_model(new_model)
+        try:
+            with connection.schema_editor() as editor:
+                editor.create_model(new_model)
+        except ProgrammingError as err:
+            # TODO: I couldn't figure out why sometimes despite the 
```

Comment:
Hmm, I'll take a look at this soon.

Reply:
@adavoudi are you able to provide steps to reproduce?

Reply:
Yeah sure, I will create an issue.